extensions/sql-database-projects/src/models/project.ts

```diff
@@ -1442,6 +1477,81 @@ export class Project implements ISqlProject {
 		}
 	}
 
+	/**
+	 * Adds a value to project property, separated by semicolon if
+	 * one or more values are already present in project property
+	 * @param propertyName Name of property
+	 * @param valueToAdd Value to be added to project property, values containing semicolon are not supported
+	 * @param caseSensitive Determines if value to add to collection should be compared against existing values case sensitively
+	 */
+	private async addValueToCollectionProjectProperty(propertyName: string, valueToAdd: string, caseSensitive?: boolean): Promise<void> {
+		if (valueToAdd.includes(';')) {
+			return;
+		}
+
+		let propertyValueToSet = this.evaluateProjectPropertyValue(propertyName);
+		if (propertyValueToSet === undefined) {
+			propertyValueToSet = valueToAdd;
+		} else {
+			const propertyValue = caseSensitive ? propertyValueToSet : propertyValueToSet.toLocaleLowerCase();
```

Comment:
so this code will convert all existing values to lowercase if a caller just one time has `caseSensitive = false`?  I'm not a fan of this sort of behind-the-scenes conversion of existing values.  IMO, if we're gonna do that, it should be done more explicitly by the caller (e.g. by the individual property that wants to normalize)

Reply:
No, that's not the case.  `propertyValue` will be set to lowercase for case insensitive comparison purposes only.
I still retain the original, case sensitive property value in `propertyValueToSet` which is used to update the project property.

Reply:
I don't think this is true. If I read the code correctly, you will do
```javascript
valueToAdd = caseSensitive ? valueToAdd : valueToAdd.toLocaleLowerCase();
```
and `valueToAdd` is later added to `propertyValueToSet` on line 1507:
```javascript
propertyValueToSet = `${propertyValueToSet};${valueToAdd}`;
```
which is ultimately stored in the project property. I think case-sensitivity flag should only affect the comparison, but not the actual value that is being stored.

Reply:
I have fixed this issue.  Now, case sensitivity only affects comparison.